cmake/linux/installer

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -e
+[ "$1" ]
+
+VERSION="$1"
+DIR="$HOME"/.lmms-$VERSION
```

Comment:
Linux users... @Umcaruje @mikobuntu @grejppi what are you opinions on a home-directory install versus system-wide?

Reply:
The Linux community needs .deb and .rpm packages.
.run is not the standard.